org.contour_terminal.Terminal.yml

```diff
@@ -0,0 +1,100 @@
+app-id: org.contour_terminal.Terminal
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
+  - --device=all          # we need access to at least DRI (for OpenGL) & PTY
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Flatpak # Required for process `flatpak-spawn --host ...`.
+
+modules:
+  - name: contour
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCONTOUR_INSTALL_TOOLS=ON"
+      - "-DCONTOUR_BLUR_PLATFORM_KWIN=ON"
+      - "-DCONTOUR_TESTING=OFF"
+      - "-DLIBTERMINAL_TESTING=OFF"
+    # NOTE: This post-install should be removed with the next release of Contour, when appdata.xml is auto-installed.
+    post-install:
+      - install -p -m644 org.contour.Terminal.appdata.xml /app/share/metainfo
```

Comment:
as I said befor **it must be installed with the right name**. The simple course of action would be for the original file to have the right name.
BTW this also install the file as `/app/share/metainfo` because the destination is a file name unless `-t/--target` is used.

Reply:
> **must be installed with the right name**

The "right name" is what [I am trying to figure out](https://github.com/flathub/flathub/pull/3349#issuecomment-1193308461) now. When that's figured out, I'll go and do the appropriate Id & file renames.

Reply:
`${FLATPAK_ID}.metainfo.xml`. That pretty clear in the documentation all over

(.appdata.xml works too)

`$FLATPAK_ID` is the shell environment that susbtitute with the app-id. Since you look at other repositories you would have noticed that too.